.flake8

```diff
@@ -62,6 +62,12 @@ ignore =
   # Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue.
   S301
 
+  # Ignore words from requests, like unprocessable
+  SC200
```

Comment:
The spell checker is annoying. However, I would rather you add those words to whitelist.dic and remove this line. There probably won't be many words to add and I have caught a lot of spelling errors thanks to this (although not so much in this project).